porch/controllers/workloadidentitybindings/config/samples/simple.yaml

```diff
@@ -15,13 +15,12 @@
 apiVersion: config.porch.kpt.dev/v1alpha1
 kind: WorkloadIdentityBinding
 metadata:
-  name: cnrm-controller-manager-example-project-id
+  name: sample-workloadidentitybinding
```

Comment:
nit: README has the other YAML inline. May be this could also be inline as well so that people can see both the configs inline all in one place.

Reply:
I considered that, but I wanted to keep the CR for the controller as a separate file to make it easy for users to just see an example, with the README providing a bit more context about how to use it.